### PR TITLE
Np 46329 bug fix redux institution changes

### PR DESCRIPTION
**Pull request by anetteOlli**

### CODE REVIEW

File: src/pages/editor/RightsRetentionStrategySettings.tsx [hunk 1]
```diff
@@ -30,10 +30,10 @@ export const RightsRetentionStrategySettings = () => {
   const updateRightsRetentionStrategy = useMutation({
     mutationFn: (customer: CustomerInstitution) => updateCustomerInstitution(customer),
     onSuccess: (response) => {
+      dispatch(setCustomer(response.data));
       dispatch(
         setNotification({ message: t('feedback.success.update_rights_retention_strategy'), variant: 'success' })
       );
-      dispatch(setCustomer(response.data));
```
Comment (anetteOlli) [inline src/pages/editor/RightsRetentionStrategySettings.tsx:36]: Could not find bug in this component, but I changed the order of these calls. Only show the success message if it also were able to update redux.

File: src/pages/editor/PublishStrategySettings.tsx [hunk 2]
```diff
@@ -53,24 +51,16 @@ export const PublishStrategySettings = () => {
...
-        dispatch(setNotification({ message: t('feedback.success.update_publish_strategy'), variant: 'success' }));
-      }
-      setIsUpdating(undefined);
-    }
-  };
+  const updateRightsRetentionStrategy = useMutation({
+    mutationFn: (customer: CustomerInstitution) => updateCustomerInstitution(customer),
+    onSuccess: (response) => {
+      dispatch(setNotification({ message: t('feedback.success.update_publish_strategy'), variant: 'success' }));
+      dispatch(setCustomer(response.data));
```
Comment (elkebab) [inline src/pages/editor/PublishStrategySettings.tsx:59]: antar dette er en glipp ref: https://github.com/BIBSYSDEV/NVA-Frontend/pull/5301/files#r1514145562
ikke at det egentlig gjør noe..

**Review by elkebab — DISMISSED** [review]
Fikk faktisk forenkla litt i samme slengen også da! 🙌

File: src/pages/editor/PublishStrategySettings.tsx (hunk 2 above)
Comment (anetteOlli) [reply src/pages/editor/PublishStrategySettings.tsx:59]: Well spotted.

Comment (anetteOlli) [reply src/pages/editor/PublishStrategySettings.tsx:58]: Endret

**Review by elkebab — APPROVED** [review]

**Pull request merged**